gm4_mountaineering/data/gm4_mountaineering/functions/climbing_effects/kill_shulkers.mcfunction

```diff
@@ -0,0 +1,7 @@
+# kill climbing shulkers
+# @s = shulker with tag gm4_mountaineering_climb_assist and needing to be killed
```

Comment:
this is quite a small function, so it's probably just better to put the commands directly in `damage_shulkers`